core/src/main/java/io/kestra/core/runners/pebble/functions/UUIDFunction.java

```diff
@@ -0,0 +1,22 @@
+package io.kestra.core.runners.pebble.functions;
+
+import io.pebbletemplates.pebble.extension.Function;
+import io.pebbletemplates.pebble.template.EvaluationContext;
+import io.pebbletemplates.pebble.template.PebbleTemplate;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+public class UUIDFunction implements Function {
+
+  @Override
+  public Object execute(
+      Map<String, Object> args, PebbleTemplate self, EvaluationContext context, int lineNumber) {
+    return UUID.randomUUID().toString().replace("-", "");
```

Comment:
Why removeing the `-` ?
With them, this is a standard UUID which is parseable if needed (I think it contains the current timestamp somewhere).

Reply:
I can leave it on if you like. Since it was an ID I assumed its better to leave it out. Will make the change as requested